components/header/index.jade

```diff
@@ -1,15 +1,18 @@
 include ../nav/index
 
 .slide-show-container
-  .header-slide.slide-1
-  .header-slide.slide-2
-  .header-slide.slide-3.active-banner
+  .header-slide.slide-1.header-slide-credit
```

Comment:
Not seeing this defined anywhere, is there any reason this class needs to be here?